scripts/check_license.py

```
@@ -0,0 +1,37 @@
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import subprocess
+import sys
+from pathlib import Path
+
+extensions = [".py", ".sh", ".go"]
+excluded_dirs = {".git", ".venv", "vendor"}
+
+ def should_include(path: Path) -> bool:
```

Ruff (0.11.9)

SyntaxError: Unexpected indentation

---

SyntaxError: Expected an indented block after function definition

---

SyntaxError: Expected a statement

---

SyntaxError: Expected an identifier, but found a keyword 'return' that cannot be used here

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

Line too long (95 > 88)

(E501)